tests/test_room.py

```diff
@@ -4,77 +4,55 @@
 import unittest
 from mock import Mock, patch
 
-class Test(unittest.TestCase):
+class TestRoom(unittest.TestCase):
 	
 	def setUp(self):
-		self.room = Room("Hogwards")
+		
+		self.room = Room("Hogwards","OFFICE")
 
 	def test_room_name_and_type(self):
 		self.assertNotEqual(self.room.Room_name, "", msg = "Cannot create empty room.")
 		self.assertIn(
```

Comment:
If you say the room is an office on line 12 then check if room type is office here, its too obvious.